frontend/src/modules/Core/Constants/index.ts

```diff
@@ -2,3 +2,10 @@ export * from '@core/Constants/Routes'
 export * from '@core/Constants/Theme'
 export * from '@core/Constants/Typography'
 export * from '@core/Constants/Media'
+
+// GENERAL CONSTANTS, FOR USE WITH FILE EXPORTING
+export const CSV_FILE = 'csv'
+export const XLX_FILE = 'xlx'
+export const PDF_FILE = 'pdf'
+export const DOWNLOAD_ALL = 'all'
+export const DOWNLOAD_NETIDS = 'netids'
```

Comment:
A better place might be in a separate TS file that's exported here instead of in index.ts directly (to follow existing convention)